### PR TITLE
Rework the OWC #2 article

### PR DESCRIPTION
**Pull request by TPGPL**
### Description

Reworked and restructured the osu! World Cup #2 article

### Status

WIP

pls help me with the tournament schedule 🙁

### CODE REVIEW

File: wiki/Tournaments/OWC/2/en.md [hunk 1]
```diff
@@ -8,347 +8,306 @@ tags:
...
+## Links
+
+- [Tournament Schedule](https://docs.google.com/document/d/1aYgninqJ6OolEvkynUsirJMpt8ZcV4Eksb7p4xuU2Jg/edit)
+- [Pre-tournament Interviews](https://docs.google.com/document/d/1xgR7cfCf1yZD1j90_ObGKccS_9OMA6rDHhYJy8uRxNE/edit)
+
+------------------------------------------
+
+## Participants
+
+**Bold users** are the team captains.
```
Comment (WalterToro) [inline wiki/Tournaments/OWC/2/en.md]: This line can be removed

File: wiki/Tournaments/OWC/2/en.md [hunk 2]
```diff
@@ -8,347 +8,306 @@ tags:
...
+| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | 1 month supporter tag |
+| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | 1 month supporter tag |
+
+## Organization
+
+The osu! World Cup #2 was run by various community members by distributing the multitude of tasks into various fields of responsibility.
+
+| Position | Members |
+| -------- | ------- |
+| Management | ![][flag_PL] [fartownik](https://osu.ppy.sh/u/56917), ![][flag_CN] [NatsumeRin](https://osu.ppy.sh/u/151679) |
```
Comment (WalterToro) [inline wiki/Tournaments/OWC/2/en.md]: Change all user links from /u/ to /users/

File: wiki/Tournaments/OWC/2/en.md [hunk 3]
```diff
@@ -8,347 +8,306 @@ tags:
...
+- [Pre-tournament Interviews](https://docs.google.com/document/d/1xgR7cfCf1yZD1j90_ObGKccS_9OMA6rDHhYJy8uRxNE/edit)
+
+------------------------------------------
+
+## Participants
+
+**Bold users** are the team captains.
+
+| | Country | Members |
+| -- |---------|---------|
```
Comment (WalterToro) [inline wiki/Tournaments/OWC/2/en.md]: Center the country names

File: wiki/Tournaments/OWC/2/en.md [hunk 4]
```diff
@@ -8,347 +8,306 @@ tags:
...
+## Mappools
 
-Group D — Republic of Korea \\ Canada \\ Vietnam \\ Hong Kong
-
-Group E — Chile \\ France \\ Portugal \\ Latvia
+### Finals
 
-Group F — Russian Federation \\ Finland \\ Malaysia \\ Uruguay
+- NoMod:
+    - [44teru-K - F.I. [Insane]](https://osu.ppy.sh/b/106033)
```
Comment (WalterToro) [inline wiki/Tournaments/OWC/2/en.md]: A reminder to do all the escaping in the article, should be `\[Insane\]`

File: wiki/Tournaments/OWC/2/en.md (hunk 1 above)
Comment (TPGPL) [reply wiki/Tournaments/OWC/2/en.md]: copied from the TWC 2016 /shrug

**Pull request merged**